packages/blade/package.json

```diff
@@ -102,7 +106,9 @@
   },
   "dependencies": {
     "@babel/runtime": "7.16.5",
-    "use-presence": "1.1.0"
+    "use-presence": "1.1.0",
+    "args-parser": "1.3.0",
+    "jscodeshift": "0.13.1"
```

Comment:
:( needed to add the deps here. 

Another way without adding any deps would be to let user run the jscodeshift directly. 
Will look something like this: 

```sh
npx jscodeshift ./INPUT_DIR -t ./node_modules/@razorpay/blade/codemods/typography-lineheight/transformers/typography-line-height.ts --ignore-pattern=**/node_modules/**
```

Reply:
I think should be fine rather than inflating dependencies? This would anyway be a one time operation by consumers

Reply:
Agree with @divyanshu013. Lets let consumers take a bit  of a heavy load here since this is a one time migration. We can add this in the changeset.